cloudant-client/src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -323,6 +325,40 @@ public MetaInformation metaInformation() {
         return replicator;
     }
 
+    /**
+     * Lists replication jobs. Includes replications created via /_replicate endpoint as well as
+     * those created from replication documents. Does not include replications which have
+     * completed or have failed to start because replication documents were malformed. Each job
+     * description will include source and target information, replication id, a history of
+     * recent event, and a few other things.
+     *
+     * @return List of replication jobs
+     */
+    public SchedulerJobsResponse schedulerJobs() {
```

Comment:
Your doc says this returns a list, so I'm expecting something like `List<SchedulerJob>`
Wouldn't it be preferable to unpack into something like that, or is there some top level meta information shared between jobs too.

Reply:
There is top level information in the response, so it can't be a `List<SchedulerJob>`. Should I change the name `SchedulerJobsResponse` to something different and/or amend the javadoc?

Reply:
I think amending the javadoc would be fine.

Reply:
No, because it still isn't a `List<SchedulerJobsRepsonse.Job>`. What we're returning is an object containing metadata and a list of replication jobs. Maybe 
`@return replication jobs information`
is sufficient here?

I think in the `SchedulerJobsResponse` class itself it is fine to say the list of jobs i.e.
```java
/**
* @return List of replication jobs
*/
public List<Job> getJobs() {
  return jobs;
}
```

etc for the other similar methods